modules/social_features/social_group/modules/social_group_secret/config/install/group.role.secret_group-anonymous.yml

```diff
@@ -0,0 +1,13 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - group.type.secret_group
+id: secret_group-anonymous
+label: Anonymous
+weight: -102
+internal: true
+audience: anonymous
+group_type: secret_group
+permissions_ui: true
```

Comment:
Not sure if relevant but I noticed this one is on `true` and for other roles it is sometimes `false`. If you think you need to change it, please do it consistently for the other roles as well.